asmcli/commands/create-mesh.sh

```diff
@@ -256,7 +261,7 @@ EOF
 has_fleet_alias() {
   local FLEET_ID; FLEET_ID="${1}"
   local REVISION; REVISION="${2}"
-  local RAW_TRUST_DOMAIN_ALIASES; RAW_TRUST_DOMAIN_ALIASES="$(retry 2 kubectl -n istio-system get configmap istio-"${REVISION}" \
+  local RAW_TRUST_DOMAIN_ALIASES; RAW_TRUST_DOMAIN_ALIASES="$(retry 2 kubectl -n istio-system get configmap "${REVISION}" \
```

Comment:
why we are using `$REVISION` here when we patch the cm with `$REV_NAME` in line 250?

Reply:
I just didn't think changing the variable name was super useful since we're passing it in, but you're right, I should probably be consistent.